backend/de.metas.contracts/src/main/java/de/metas/contracts/modular/computing/salescontract/storagecost/ShipmentLineLog.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * #%L
+ * de.metas.contracts
+ * %%
+ * Copyright (C) 2024 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+package de.metas.contracts.modular.computing.salescontract.storagecost;
+
+import de.metas.contracts.modular.ModularContractService;
+import de.metas.contracts.modular.invgroup.interceptor.ModCntrInvoicingGroupRepository;
+import de.metas.contracts.modular.workpackage.impl.AbstractStorageCostLogHandler;
+import de.metas.lang.SOTrx;
+import de.metas.organization.LocalDateAndOrgId;
+import de.metas.util.Check;
+import lombok.NonNull;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.springframework.stereotype.Component;
+
+@Component
+class ShipmentLineLog extends AbstractStorageCostLogHandler
+{
+	public ShipmentLineLog(
+			@NonNull final ModularContractService modularContractService,
+			@NonNull final ModCntrInvoicingGroupRepository modCntrInvoicingGroupRepository,
+			@NonNull final SalesStorageCostComputingMethod computingMethod)
+	{
+		super(modularContractService, modCntrInvoicingGroupRepository, computingMethod);
+	}
+
+	@Override
+	protected Integer computeStorageDays(
+			final @NotNull CreateLogRequest createLogRequest,
+			final @NonNull LocalDateAndOrgId transactionDate,
+			final @Nullable LocalDateAndOrgId physicalClearanceDate)
+	{
+		Check.assumeNotNull(physicalClearanceDate, "Physical Clearance Date shouldn't be null ()");
+		final int daysBetween = (int)LocalDateAndOrgId.daysBetween(physicalClearanceDate, transactionDate);
```

Comment:
Please, find out if they don't use the daysBetween360 here as well. For more info, please check org.compiere.util.TimeUtil#getDaysBetween360(java.time.ZonedDateTime, java.time.ZonedDateTime) and its usages.

Reply:
This is interest specific https://en.comparis.ch/hypotheken/hypothekarzinsen/internationale-zinsusanz . And I only know this in relation to interest.
I don't expect free storageCosts days in 31days months and couldn't find anything in this regard

Reply:
OK. Just to be sure, maybe we should specify this in the UAT, that the storage days are computed naturally, not like the interest days.